index.html

```diff
@@ -4,7 +4,7 @@
     <title>Fedora API Specification</title>
     <meta charset='utf-8'>
     <meta name="description" content="Fedora API Specification">
-    <script src='https://www.w3.org/Tools/respec/respec-w3c-common'
+    <script src='http://fedora.info/spec/respec/v14.0.13/respec-w3c-common'
```

Comment:
Obviously, this depends first making sure that the server at `fedora.info` supports HTTPS connections; otherwise, you would be better off just using a relative link (omitting the hostname) to the javascript file being added to this repository.